tests/core/hexadecimal-utils/test_encode_and_decode.py

```diff
@@ -7,7 +7,7 @@
     strategies as st,
 )
 
-from eth.utils.hexadecimal import (
+from eth_utils import (
```

Comment:
See comment on import order further down. This one should be before the `hypothesis` import (alphabetically within 3rd party imports)